services/ui-src/src/components/tables/EntityStatusIcon.tsx

```diff
@@ -1,65 +1,53 @@
 // components
 import { Box, Image, Text } from "@chakra-ui/react";
 // assets
-import unfinishedIcon from "assets/icons/icon_error_circle_bright.png";
-import unfinishedIconDark from "assets/icons/icon_error_circle.png";
+import unfinishedIcon from "assets/icons/icon_error_circle.png";
 import successIcon from "assets/icons/icon_check_circle.png";
-import successIconDark from "assets/icons/icon_check_circle_dark.png";
 import closedIcon from "assets/icons/icon_circle-minus-gray.png";
+// types
+import { EntityStatuses } from "types";
 
-export enum EntityStatuses {
-  COMPLETE = "complete",
-  CLOSE = "close",
-  NO_STATUS = "no status",
-  DISABLED = "disabled",
-}
-
-export type EntityStatusType = EntityStatuses | boolean | undefined;
+type EntityStatusType = EntityStatuses | undefined;
 
-export const EntityStatusIcon = ({ entityStatus, isPdf }: Props) => {
+export const EntityStatusIcon = ({ entityStatus, showLabel }: Props) => {
   const statusIcon = (status: EntityStatusType) => {
     switch (status) {
-      case true:
       case EntityStatuses.COMPLETE:
         return {
-          src: isPdf ? successIconDark : successIcon,
-          alt: isPdf ? "" : "complete icon",
+          src: successIcon,
+          alt: showLabel ? "" : "complete icon",
           style: sx.successText,
           text: "Complete",
         };
       case EntityStatuses.CLOSE:
         return {
-          src: isPdf ? closedIcon : closedIcon,
-          alt: isPdf ? "" : "close icon",
+          src: closedIcon,
+          alt: showLabel ? "" : "close icon",
           style: sx.closeText,
           text: "Closed",
         };
-      case EntityStatuses.NO_STATUS:
-      case EntityStatuses.DISABLED:
-        return undefined;
-      default:
+      case EntityStatuses.INCOMPLETE:
         return {
-          src: isPdf ? unfinishedIconDark : unfinishedIcon,
-          alt: isPdf ? "" : "warning icon",
+          src: unfinishedIcon,
+          alt: showLabel ? "" : "warning icon",
           style: sx.errorText,
           text: "Error",
         };
+      case EntityStatuses.NO_STATUS:
```

Comment:
Agreed; I think we can just let default handle these cases.

Reply:
I do think it's nice to define these, given it's an enum, just so that it's clear to other developers those situations are acknowledged and accounted for in the switch statement, even if they are ignored.
I'm ok with removing them if you or others feel strongly about it though